### DISCUSSION TITLE
Extracting bounding boxes for characters rather than words

### DISCUSSION THREAD

**Discussion by Gittingthehubbing** [mindee/doctr#1157] [Q&A]
Hi,

thanks for all your great work.

I was wondering if there was a way to perform character level OCR using doctr. It seems to work really well on getting the bounding boxes for each word but I can't seem to find a way to get the boxes for each character.

Is there a way to do this?

Thanks in advance

**Comment by Mayur28** [comment]
Hi,

Have you managed to find a solution? I would like to extract bounding boxes for characters as well.

**Comment by Gittingthehubbing** [comment]
Hi,

Unfortunately not. I've been using Tesseract to do it which works ok but is
definitely not perfect.

**Comment by Mayur28** [comment]
I've been using tesseract as well and as you mentioned, the performance is
not great, but the performance improves slightly if you apply image
processing techniques. Haven't found any other promising solutions as yet.
